packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -125,12 +143,24 @@ export default defineComponent<KottiPopover.PropsInternal>({
 		return {
 			close,
 			contentRef,
+			formKey: computed(() => (formContext === null ? null : 'NONE')),
```

Comment:
have you tested this in action?

Reply:
yes
it works

Reply:
We could actually make this a util function that we export as well. It could be useful also for users of kotti.

Reply:
it's kind of an anti-recommendation though :) 

We don't expect users to hack around the form usage. Either a field is used within a form or it's not.. 

You're probably already familiar with the form-within-form hack we have on our user-app and how it's not a good thought in the first place